### PR TITLE
Increment GCM IV

### PR DESCRIPTION
**Pull request by olorin**
! @markhibberd @erikd-ambiata @thumphries

### CODE REVIEW

File: test/Test/Tinfoil/AEAD/AESGCM/Iv.hs [hunk 1]
```diff
@@ -31,6 +31,11 @@ prop_incrementInvocationField_overflow f =
     Right (InvocationField _ (InvocationCount x)) ->
       x =/= 0
 
+
+prop_incrementGcmIv :: GcmIv -> Property
+prop_incrementGcmIv iv =
+  (incrementGcmIv iv) =/= (Right iv)
+
```
Comment (erikd-ambiata) [inline test/Test/Tinfoil/AEAD/AESGCM/Iv.hs]: LISP anyone?

**Comment by erikd-ambiata** [comment]
:-1:

**Comment by thumphries** [comment]
🚍

File: test/Test/Tinfoil/AEAD/AESGCM/Iv.hs (hunk 1 above)
Comment (olorin) [inline test/Test/Tinfoil/AEAD/AESGCM/Iv.hs]: I like LISP.

**Comment by markhibberd** [comment]
👍

**Pull request merged**